sdk/storage/storage-file-share/src/generatedModels.ts

```diff
@@ -52,5 +52,6 @@ export {
   SignedIdentifier as SignedIdentifierModel,
   SourceModifiedAccessConditions,
   FileForceCloseHandlesHeaders,
-  DirectoryForceCloseHandlesHeaders
+  DirectoryForceCloseHandlesHeaders,
+  CopyFileSmbInfo,
```

Comment:
Is `review/storage-file-share.api.md` modified, after exporting these new types? I don't see the api.md file in this PR.

Reply:
will merge all the changes in this release and send a separate PR for API chagnes.